src/styles/main.scss

```diff
@@ -1,3 +1,239 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+$colorLink: #00acdc;
+$colorText: #060b35;
+
+* {
+  box-sizing: border-box;
   margin: 0;
 }
+
+.body {
+  color: $colorText;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
```

Comment:
create separated css files for each BEM block, like `.header`, `.stars` etc (header.css etc)
separated for common styles, like `*` and `body` (common.css or main.css)
and separated for variables (variables.css)

After doing that - import those files to entry file (index.css) that you will link to your html page